pyaedt/hfss.py

```diff
@@ -2365,7 +2378,7 @@ def validate_full_design(self, dname=None, outputdir=None, ports=None):
                 val_list.extend(temp)
             os.remove(temp_val_file)
         else:
-            msg = "** No design validation file isfound. **"
+            msg = "** No design validation file is found. **"
```

Comment:
@maxcapodi78 I am assuming it won't cause any deprecation issue here as this error message is appended to the list of validation message returned.

Reply:
Correct, same would go with changes within any methods that do not affect the public API.